templates/our-project-contents.html

```diff
@@ -0,0 +1,10 @@
+<div class=" col-md-6 animate__animated animate__fadeInUp">
+    <a  data-link="url" add target="_blank" class="col-lg-12">
```

Comment:
why do you have `add target="_blank"` in here?

Reply:
used data-href instead of data-link and used target="_blank"